requirements.txt

```diff
@@ -12,4 +12,4 @@ cosl ==0.0.15
 # juju 3.1.2.0 depends on pyyaml<=6.0 and >=5.1.2
 PyYAML ==6.0.*
 pyOpenSSL==24.2.1
-github_runner_manager @ git+https://github.com/canonical/github-runner-manager.git@33bbaff42d7cc0f250006fdd08d24659cef364c9
+github_runner_manager @ git+https://github.com/canonical/github-runner-manager.git@feat/reactive-cos-ISD-1537
```

Comment:
Need to fix this before merge

Reply:
Sure, which is why I still mark this PR as a draft until https://github.com/canonical/github-runner-manager/pull/21 gets merged.